### PR TITLE
#1047 - Add docker support

### PR DESCRIPTION
**Pull request by jcklie**
**What's in the PR**
* Docker file construction

**How to test manually**
* Run `mvn -Pdocker clean install docker:build` and then start a container of this type `docker run -v /path/on/host/inception/repository:/export -p port-on-host:8080 inceptionproject/inception-snapshots:latest`
* Run `mvn -Pdocker clean install docker:build -Ddocker.image.name="inceptionproject/inception"` and then start a container of this type `docker run -v /path/on/host/inception/repository:/export -p port-on-host:8080 inceptionproject/inception:latest`
* In both cases, check that INCEpTION started by going to `localhost:8080`

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [X] PR updates documentation

### CODE REVIEW

**Comment by jcklie** [comment]
Docker file itself works, the compose not yet.

**Comment by reckart** [comment]
We should include the relevant parts of the admin documentation as well.

File: inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_docker.adoc [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+== Quick start
+
+If you have Docker installed, you can run INCEpTION using
+
+[source,text,subs="+attributes"]
+----
+docker run -it --name inception -p8080:8080 inception/inception:{revnumber}
+----
+
+The command download INCEpTION from Dockerhub and starts it on port 8080. If this port is not
```
Comment (UWinch) [inline inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_docker.adoc]: Typo: downloads

**Comment by jcklie** [comment]
It uses the version that is in the pom, so it should be snapshot in your case.

**Comment by reckart** [comment]
I would be great if we could cherry-pick the dockerization into the 0.8.x branch!

**Comment by jcklie** [comment]
@reckart I cherry pick once this is merged

**Pull request merged**